email_service/tests/main_test.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from fastapi.testclient import TestClient
+from unittest.mock import patch
+from email_service.app.main import email_app
+from fastapi_mail import FastMail
+client = TestClient(email_app)
+
+def test_send_activation_email_success():
```

Comment:
this and test_send_password_reset_email_success  was not passing, i haven't got a chance to look into it, feel free to take a look and propose any change for updating tests or code

Reply:
```suggestion
response = client.post(
        "/send-activation-email",
        params={"token": "fake_token"}
    )
```
Since the request parameters don't have a schema, json={"token": "fake_token"} couldn't function properly, so I changed it to params={"token": "fake_token"}.

Reply:
```suggestion
return {"status": "200", "message": "Password reset email sent"}
```
Because response_model=schemas.Message, I modified the return to conform to the Message schema.